packages/react/src/sx.ts

```diff
@@ -29,6 +29,11 @@ export interface SxProp {
   sx?: BetterSystemStyleObject
 }
 
+export interface DeprecatedSxProp {
```

Comment:
Adding `DeprecatedSxProp` for other components to use in the batch: https://github.com/github/primer/issues/4261